doc/api/deprecations.md

```diff
@@ -1795,6 +1795,12 @@ Type: End-of-Life
 `runInAsyncIdScope` doesn't emit the `'before'` or `'after'` event and can thus
 cause a lot of issues. See <https://github.com/nodejs/node/issues/14328>.
 
+<a id="DEP0087"></a>
+### DEP0087: _Unassigned deprecation_
+
+<a id="DEP0088"></a>
+### DEP0088: _Unassigned deprecation_
+
```

Comment:
Maybe we shouldn't error if there are missing numbers. It becomes an issue with release lines because we do not always backport deprecations and that can leave gaps.

Reply:
Hum right, I haven't thought of that... What about using a comment to explicitly skip them? 

```suggestion
<!-- md-lint skip-deprecation DEP0087 -->
<!-- md-lint skip-deprecation DEP0088 -->

```